.circleci/config.yml

```diff
@@ -36,6 +36,31 @@ aliases:
       command: |
         sudo apt-get update
         sudo apt-get install -y libcairo2-dev
+  - &check_versions
+    run:
+      name: Check that all versions are the same if $CIRCLE_TAG is defined
+      command: |
+        if [[ ! -z ${CIRCLE_TAG+x} ]]; then
+          python -m pip install --upgrade setuptools_scm
+          BACKEND_VERSION=$(python -m setuptools_scm)
```

Comment:
I feel this will not work as `python -m setuptools_scm` should return one version above the tag. 🤔